.github/workflows/hil_test_zephyr.yml

```diff
@@ -81,13 +85,13 @@ jobs:
         run: |
           pip install pytest pytest-timeout
           pip install tests/hil/scripts/pytest-hil
-          pip install git+https://github.com/golioth/python-golioth-tools@v0.5.1
+          pip install git+https://github.com/golioth/python-golioth-tools@f6bd62aaf239409c57b2a0ccbb65c2df0833564f
```

Comment:
Point to v0.6.0 after https://github.com/golioth/python-golioth-tools/pull/29 is merged